Solutions/SecurID/Data Connectors/RSASecurID.json

```diff
@@ -0,0 +1,118 @@
+{
+    "id": "RSASecurIDAM",
+    "title": "RSA® SecurID (Authentication Manager)",
+    "publisher": "RSA",
+    "descriptionMarkdown": "The [RSA® SecurID Authentication Manager](https://www.securid.com/) data connector provides the capability to ingest [RSA® SecurID Authentication Manager events](https://community.rsa.com/t5/rsa-authentication-manager/log-messages/ta-p/571404) into Azure Sentinel. Refer to [RSA® SecurID Authentication Manager documentation](https://community.rsa.com/t5/rsa-authentication-manager/getting-started-with-rsa-authentication-manager/ta-p/569582) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**RSASecurIDAMEvent**](https://aka.ms/sentinel-rsasecuridam-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "RSASecurIDAM",
+        "baseQuery": "RSASecurIDAMEvent"
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Sources",
+        "query": "RSASecurIDAMEvent\n | summarize count() by tostring(DvcHostname)\n | top 10 by count_"
+    }],
+    "dataTypes": [{
+        "name": "Syslog(RSASecurIDAMEvent)",
```

Comment:
There should be a space after Syslog. 
Syslog (RSASecurIDAMEvent)

Reply:
updated